src/bp/ui-studio/src/web/index.html

```diff
@@ -1,25 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
-    <meta name="description" content="" />
-    <meta name="keywords" content="" />
-    <script src="env.js"></script>
-    <script>
-      window.ROOT_PATH = ''
-    </script>
-    <base href="/" />
-    <link rel="icon" type="image/png" href="assets/ui-studio/public/img/favicon-32x32.png" sizes="32x32" />
-    <link rel="icon" type="image/png" href="assets/ui-studio/public/img/favicon-16x16.png" sizes="16x16" />
-    <link href="assets/ui-studio/public/external/material-icons.css" rel="stylesheet" />
-    <link href="assets/ui-studio/public/external/font-lato.css" rel="stylesheet" />
-    <link href="assets/ui-studio/public/external/bootstrap.min.css" rel="stylesheet" />
-    <link href="assets/custom-theme.css" rel="stylesheet" />
-    <link href="assets/modules/channel-web/default.css" rel="stylesheet" />
-  </head>
 
-  <body>
-    <div id="app"></div>
-  </body>
-</html>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+	<meta name="description" content="" />
+	<meta name="keywords" content="" />
+	<script src="env.js"></script>
+	<script>
+		window.ROOT_PATH = ''
+	</script>
+	<base href="/" />
+	<link rel="icon" type="image/png" href="assets/ui-studio/public/img/favicon-32x32.png" sizes="32x32" />
+	<link rel="icon" type="image/png" href="assets/ui-studio/public/img/favicon-16x16.png" sizes="16x16" />
+	<link href="assets/ui-studio/public/external/material-icons.css" rel="stylesheet" />
+	<link href="assets/ui-studio/public/external/font-lato.css" rel="stylesheet" />
+	<link href="assets/ui-studio/public/external/font-roboto.css" rel="stylesheet" />
+	<link href="assets/ui-studio/public/external/bootstrap.min.css" rel="stylesheet" />
+	<link href="assets/custom-theme.css" rel="stylesheet" />
+	<link href="assets/modules/channel-web/default.css" rel="stylesheet" />
+</head>
+
+<body>
+	<div id="app"></div>
+</body>
+
+</html>
```

Comment:
Please validate your vscode configuration, this indentation is definitely not right. It should be two spaces, and body should be indented